src/index.js

```diff
@@ -18,6 +19,7 @@ let reduxStore = {
     risks: risks,
     asn: asn,
     cubeByRiskByCountry: {},
+    cubeByRiskByASN: {},
     cubeByRiskByAS: tempData,
```

Comment:
What's difference here? 
`cubeByRiskByASN: {},` and `cubeByRiskByAS: tempData`
Do we actually need second one for build?